dist.java/java/index.html

```diff
@@ -3,7 +3,7 @@
 
   <head>
     <title>openshift-jvm</title>
-    <base href="/">
+    <base href="/dev-console/java/">
```

Comment:
You need to reset this to `/` when committing. It will be set correctly by the server when index.html is served using the real context root.